src/idlcxx/src/types.c

```diff
@@ -45,16 +45,26 @@ emit_member(
   else
     type_spec = idl_type_spec(node);
 
+  const idl_member_t *mem = idl_parent(node);
+  assert(idl_is_member(mem));
+
   name = get_cpp11_name(node);
   if (IDL_PRINTA(&type, get_cpp11_type, type_spec, gen) < 0)
     return IDL_RETCODE_NO_MEMORY;
   type_spec = idl_strip(type_spec, IDL_STRIP_ALIASES | IDL_STRIP_FORWARD);
-  if (idl_is_array(type_spec))
+  value = NULL;
+  if (idl_is_array(type_spec)) {
     value = "{ }";
-  else if (!idl_is_enum(type_spec) && !idl_is_base_type(type_spec))
```

Comment:
I get a conflict here because of the `bitmask` that appears straightforward to resolve, however ...